rasa/importers/multi_project.py

```diff
@@ -162,13 +164,39 @@ def _is_in_imported_paths(self, path) -> bool:
 
     def add_import(self, path: Text) -> None:
         self._imports.append(path)
-
+    
+    def _log_merge_infos(self, domains) -> None:
+        merges = {
```

Comment:
I am not entirely sure if it would work, but I think we could just do `Domain.empty().as_dict()` instead of repeating the Domain structure here.

Reply:
Unfortunately not since `merges` is not exactly like the Domain structure but just here to store infos about actions/entities/forms/intents/slots about to be merged.

For info, here is what `Domain.empty().as_dict()` provides:

```
{'config': {'store_entities_as_slots': True}, 'session_config': {'session_expiration_time': 0, 'carry_over_slots_to_new_session': True}, 'intents': [], 'entities': [], 'slots': {}, 'responses': {}, 'actions': [], 'forms': []}
```